x/emissions/migrations/v7/migrate.go

```diff
@@ -0,0 +1,114 @@
+		MinTopicWeight:                      oldParams.MinTopicWeight,
+		RequiredMinimumStake:                oldParams.RequiredMinimumStake,
+		RemoveStakeDelayWindow:              oldParams.RemoveStakeDelayWindow,
+		MinEpochLength:                      oldParams.MinEpochLength,
+		BetaEntropy:                         oldParams.BetaEntropy,
+		LearningRate:                        oldParams.LearningRate,
+		MaxGradientThreshold:                oldParams.MaxGradientThreshold,
+		MinStakeFraction:                    oldParams.MinStakeFraction,
+		MaxUnfulfilledWorkerRequests:        oldParams.MaxUnfulfilledWorkerRequests,
+		MaxUnfulfilledReputerRequests:       oldParams.MaxUnfulfilledReputerRequests,
+		TopicRewardStakeImportance:          oldParams.TopicRewardStakeImportance,
+		TopicRewardFeeRevenueImportance:     oldParams.TopicRewardFeeRevenueImportance,
+		TopicRewardAlpha:                    oldParams.TopicRewardAlpha,
+		TaskRewardAlpha:                     oldParams.TaskRewardAlpha,
+		ValidatorsVsAlloraPercentReward:     oldParams.ValidatorsVsAlloraPercentReward,
+		MaxSamplesToScaleScores:             oldParams.MaxSamplesToScaleScores,
+		MaxTopInferersToReward:              oldParams.MaxTopInferersToReward,
+		MaxTopForecastersToReward:           oldParams.MaxTopForecastersToReward,
+		MaxTopReputersToReward:              oldParams.MaxTopReputersToReward,
+		CreateTopicFee:                      oldParams.CreateTopicFee,
+		GradientDescentMaxIters:             oldParams.GradientDescentMaxIters,
+		RegistrationFee:                     oldParams.RegistrationFee,
+		DefaultPageLimit:                    oldParams.DefaultPageLimit,
+		MaxPageLimit:                        oldParams.MaxPageLimit,
+		MinEpochLengthRecordLimit:           oldParams.MinEpochLengthRecordLimit,
+		BlocksPerMonth:                      oldParams.BlocksPerMonth,
+		PRewardInference:                    oldParams.PRewardInference,
+		PRewardForecast:                     oldParams.PRewardForecast,
+		PRewardReputer:                      oldParams.PRewardReputer,
+		CRewardInference:                    oldParams.CRewardInference,
+		CRewardForecast:                     oldParams.CRewardForecast,
+		CNorm:                               oldParams.CNorm,
+		EpsilonReputer:                      oldParams.EpsilonReputer,
+		HalfMaxProcessStakeRemovalsEndBlock: oldParams.HalfMaxProcessStakeRemovalsEndBlock,
+		EpsilonSafeDiv:                      oldParams.EpsilonSafeDiv,
+		DataSendingFee:                      oldParams.DataSendingFee,
+		MaxElementsPerForecast:              oldParams.MaxElementsPerForecast,
+		MaxActiveTopicsPerBlock:             oldParams.MaxActiveTopicsPerBlock,
+		MaxStringLength:                     oldParams.MaxStringLength,
+		InitialRegretQuantile:               oldParams.InitialRegretQuantile,
+		PNormSafeDiv:                        oldParams.PNormSafeDiv,
+		GlobalWhitelistEnabled:              oldParams.GlobalWhitelistEnabled,
```

Comment:
above the comments say that the global whitelist enabled will be flipped on ... but here it seems to use the old value rather than forcing it to true?

does the comment just need to be fixed or does this migration for 0.8.0 emissions version 7 have some intended change to the whitelists on testnet? cc: @kpeluso

Reply:
Those params come from previous version, don't they? 
The flipping was done on `v0_7_0` migration package. This is `v0_8_0`. 

I think the only params from this version are those two re. outlier detection. Please shout otherwise
```
// NEW PARAMS
InferenceOutlierDetectionThreshold: defaultParams.InferenceOutlierDetectionThreshold,
InferenceOutlierDetectionAlpha:     defaultParams.InferenceOutlierDetectionAlpha,
```